modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -215,3 +215,258 @@
         summary: Smoke-mini is unavailable
         description: |
           Smoke-mini is unavailable. Please check the linked alerts to find the real problem.
+
+- name: d8.upmeter.resources
+  rules:
+    - alert: D8UpmeterProbeGarbageConfigmap
+      expr: |
```

Comment:
Let's set the `for` at least for 5 minutes or more. We do not need to react if configmaps number is back to normal.